src/service/viewport/viewport-binding-ios-embed-sd.js

```diff
@@ -404,6 +404,24 @@ export class ViewportBindingIosEmbedShadowRoot_ {
         + this.getBorderTop();
   }
 
+  /** @override */
+  contentHeightChanged() {
+    if (isExperimentOn(this.win, 'scroll-height-bounce')) {
+      // Refresh the overscroll (`-webkit-overflow-scrolling: touch`) to avoid
+      // iOS rendering bugs. See #8798 for details.
+      this.vsync_.mutate(() => {
+        setImportantStyles(this.scroller_, {
+          '-webkit-overflow-scrolling': 'auto',
+        });
+        this.vsync_.mutate(() => {
+          setImportantStyles(this.scroller_, {
+            '-webkit-overflow-scrolling': 'touch',
+          });
```

Comment:
Won’t this interrupt active scrolling? Probably need to wait until we’re not scrolling before applying.

Reply:
This is triggered from resources path that waits for no or slow scrolling and thus this shouldn't be a concern.

Reply:
This is also triggered by element change size. Maybe check that relayoutAll is set?

Reply:
`relayoutAll` is actually normally all good. That simply means that the whole viewport is resized and full relayout/rerendering expected. Element change size is actually in scope - this is why amp-list resize borks up scrolling.